cmd/power/power.go

```diff
@@ -19,6 +21,53 @@ const (
 	DOCKER_CGROUPS_FILE = "/proc/self/cgroup"
 )
 
+func PowerMain() {
+	app := cli.NewApp()
+
+	app.Name = os.Args[0]
+	app.Version = config.VERSION
+	app.Author = "Rancher Labs, Inc."
+	app.Email = "sid@rancher.com"
+	app.EnableBashCompletion = true
+	app.Action = power
+	app.HideHelp = true
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "f",
+			Usage: "Does not invoke shutdown, and instead performs the actual action",
+		},
+		cli.BoolFlag{
+			Name:  "n",
```

Comment:
The docs here talk about linux always calling sync http://manpages.ubuntu.com/manpages/gutsy/man8/halt.8.html